src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
-class PageObject {
+class PageObject extends Header {
+  header = new Header();
```

Comment:
Move `header` property assignment inside the PageObject constructor. Example: `constructor(url) { this.header = new Header(); }`